### PR TITLE
Updated FIT.pm and fit2tcx.pl

### PR DESCRIPTION
**Pull request by patjoly**
With Kiyokazu Suto's permission, I have added FIT.pm and fit2tcxp.pl to Perl's CPAN repository. I am now  maintaining the module and since updated it to the latest FIT profile as per the Garmin SDK.

The namespace was changed to Geo:: as I also maintain modules for GPX and TCX files within the same namespace.

Changes made to the following files:

plugins/garmin-fit/bin/Geo/FIT.pm
    . renamed from plugins/garmin-fit/bin/Garmin/FIT.pm
    . updated to CPAN version 1.09 (https://metacpan.org/pod/Geo::FIT)
    . up-to-date with Garmin FIT SDK profile 21.107

plugins/garmin-fit/bin/fit2tcx.pl
    . renamed from plugins/garmin-fit/bin/fit2tcx (had no extension)
    . updated to CPAN version 1.09 (https://metacpan.org/pod/Geo::FIT)

imports/file_garminfit.py
    . updated to reflect the renamed script

setup.py
    . updated to reflect the 2 renamed files above

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ build
 __pycache__
 .eggs/
 pytrainer.egg-info
+Notes
+Todo
```
Comment (viiru-) [inline .gitignore:9]: I'm not going to merge this commit. If you want to keep personal files in the repo and ignore them please use .git/info/exclude instead.

**Comment by viiru-** [comment]
Also the renames need to happen in a separate commit from the updates (so that git log can show the history as it happened).

File: .gitignore (hunk 1 above)
Comment (patjoly) [reply .gitignore:9]: Done. Submitted a new PR (#238) that splits the commit into 2. You can ignore/cancel this one.

**Comment by patjoly** [comment]
Looks like my reply to your 

> Also the renames need to happen in a separate commit from the updates (so that git log can show the history as it happened).

Done, I submitted a new PR(#238) splitting the commit into 2. You can ignore/cancel this one.

**Comment by viiru-** [comment]
> Done, I submitted a new PR(#238) splitting the commit into 2. You can ignore/cancel this one.

There was no need to submit a new PR, you could have just modified this one. But as that is now done I'll close this one..

**Pull request closed without merging**